lib/script-loader.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Overrides the script-loader.php file.
+ *
+ * @package gutenberg
+ */
+
+// Remove core actions to override.
+remove_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles' );
+remove_action( 'wp_footer', 'wp_enqueue_global_styles', 1 );
+
+/**
+ * Enqueues the global styles defined via theme.json.
+ *
+ * Copy of the core `wp_enqueue_global_styles`. Uses helper methods bundled with the plugin.
+ *
+ * @return void
+ */
+function gutenberg_enqueue_global_styles() {
```

Comment:
This brings this function back after having being removed at https://github.com/WordPress/gutenberg/pull/50079